src/PokemonGymOwner.java

```diff
@@ -0,0 +1,32 @@
+import java.util.List;
+
+public class PokemonGymOwner extends PokemonTrainer {
+    private final String name;
+    private final String town;
+    List<Pokemon> pokemons;
+
+    public PokemonGymOwner(String name, String town) {
+        super(name);
```

Comment:
Je ziet hier al dat je zowel de name van de super als de name van deze kalsse zet. Dat is dubbelop en dus onnodig. Je mag in deze klasse de name weg laten. Je kunt de name van de super gebruiken.